x-pack/plugins/endpoint/public/plugin.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin, CoreSetup } from 'kibana/public';
+
+export class EndpointPlugin implements Plugin<{}, {}> {
+  public setup(core: CoreSetup) {
+    core.application.register({
+      id: 'endpoint',
+      title: 'Endpoint',
```

Comment:
I think we probably want this translated as well.  I notice the Discover title is
```

import { i18n } from '@kbn/i18n';
... 
        title: i18n.translate('kbn.discoverTitle', {
            defaultMessage: 'Discover',
          }),
```

Reply:
Are proper-nouns like product and feature names usually translated?

Reply:
I _think_ they should be.  I googled "Pepsi" in Chinese and something comes up, and some random website says 

```
Route 1: Keep in the source language
As a general rule, if the source script and target script are different (Arabic to English, or Thai to English, for example), no words should be left in the source script in your translation"
```

As far as company policy though, I don't know.  I would guess over translating is better than under translating.  cc @Bamieh - if you have any additional insight.